README.md

```diff
@@ -1,2 +1,107 @@
-# neon-datasets
-Repository for code and derived data from NEON data products
+## Curated NEON datasets
+
+Contains scripts for downloading and cleaning data, and the resulting data files. 
+Metadata for original and curated datasets are in this README. 
+
+### 1. Plant cover
+
+#### Original data
+
+- **Plant presence and percent cover** dataset 
```

Comment:
some useful information here, but not sure if the columns of the raw data are required.

Reply:
Removed those [here](https://github.com/genophenoenvo/neon-datasets/pull/1/commits/71ca2972549807254bd4fd1c155ba0468b49c654).